src/views/edm/EDMBenchmarkChart.vue

```diff
@@ -378,6 +385,7 @@ export default {
     }),
 
     update() {
+      console.log('Updating EDM Benchmark Chart...')
```

Comment:
Remove any `console.log`, it only used for testing and should be removed on the production environment.